flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/JdbcTableSourceITCase.java

```diff
@@ -143,20 +149,19 @@ public void testProjectableJdbcSource() throws Exception {
 				")"
 		);
 
-		StreamITCase.clear();
-		tEnv.toAppendStream(tEnv.sqlQuery("SELECT timestamp6_col, decimal_col FROM " + INPUT_TABLE), Row.class)
-				.addSink(new StreamITCase.StringSink<>());
-		env.execute();
+		TableResult tableResult = tEnv.executeSql("SELECT timestamp6_col, decimal_col FROM " + INPUT_TABLE);
+
+		List<String> results = manifestResults(tableResult);
 
-		List<String> expected =
-			Arrays.asList(
-				"2020-01-01T15:35:00.123456,100.1234",
-				"2020-01-01T15:36:01.123456,101.1234");
-		StreamITCase.compareWithList(expected);
+		assertThat(
+				results,
+				containsInAnyOrder(
+						"2020-01-01T15:35:00.123456,100.1234",
```

Comment:
@aljoscha for the future: please use instances instead of strings

Reply:
Noted, that's a very good point.